validation_server/validation_server.js

```diff
@@ -0,0 +1,82 @@
+const axios = require("axios");
+const {
+  HttpTraceContext,
+  getExtractedSpanContext,
+
+  setExtractedSpanContext,
+} = require("@opentelemetry/core");
+const { BasicTracerProvider } = require("@opentelemetry/tracing");
+const { Context } = require("@opentelemetry/context-base");
+const bodyParser = require("body-parser");
+const {
+  defaultGetter,
+  defaultSetter,
+  propagation,
+  trace,
+} = require("@opentelemetry/api");
+
+// set global propagator
+const myPropagator = new HttpTraceContext();
+propagation.setGlobalPropagator(myPropagator);
+
+// Create a provider for activating and tracking spans
+const tracerProvider = new BasicTracerProvider({});
+
+// Register the tracer
+tracerProvider.register();
+
+// Get a tracer
+const tracer = trace.getTracer("w3c-tests");
+
+// --- Simple Express app setup
+const express = require("express");
+const port = 5000;
+const app = express();
+
+app.use(bodyParser.json());
+
+// Mount our demo route
+app.post("/verify-tracecontext", (req, res) => {
+  const context = myPropagator.extract(
```

Comment:
Use the api here, not the propagator directly.

`propagation.extract(req.headers)`